master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -464,13 +464,14 @@ private[celeborn] class Master(
     if (worker == null) {
       logWarning(s"Unknown worker $host:$rpcPort:$pushPort:$fetchPort:$replicatePort" +
         s" for WorkerLost handler!")
-      return
-    }
-
-    statusSystem.handleWorkerLost(host, rpcPort, pushPort, fetchPort, replicatePort, requestId)
-
-    if (context != null) {
-      context.reply(WorkerLostResponse(true))
+      if (context != null) {
+        context.reply(WorkerLostResponse(false))
```

Comment:
It's ambiguous when reply(WorkerLostResponse with false), how about keep replay true( or false with code-UnKnow).

Reply:
Change to true.